src/chainparams.cpp

```diff
@@ -92,8 +92,8 @@ class CMainParams : public CChainParams {
 
         // Deployment of Taproot
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 1199145601; // January 1, 2008
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = 1230767999; // December 31, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1199145601; // January 1, 2008. Placeholder. Taproot BIP9 params TBD.
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1230767999; // December 31, 2008. Placeholder. Taproot BIP9 params TBD.
```

Comment:
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 0; // Never / undefined. Placeholder. Taproot BIP9 params TBD.
```

I don't think BIP 9 allows overlapping times on the same bit?

Reply:
Sorry, I was confused. The testdummy bit is different from the taproot bit, so should be fine. Though I'd prefer to still set this to `0` (like segwit did) unless there is reason to do it differently.

Reply:
I'm happy either way. Can change to zero if that's what others want.